google-cloud-recommendations-ai/src/main/java/com/google/cloud/recommendationengine/v1beta1/stub/UserEventServiceStubSettings.java

```diff
@@ -320,12 +320,7 @@ public String extractNextToken(ListUserEventsResponse payload) {
     static {
       ImmutableMap.Builder<String, ImmutableSet<StatusCode.Code>> definitions =
           ImmutableMap.builder();
-      definitions.put(
-          "idempotent",
-          ImmutableSet.copyOf(
-              Lists.<StatusCode.Code>newArrayList(
-                  StatusCode.Code.DEADLINE_EXCEEDED, StatusCode.Code.UNAVAILABLE)));
-      definitions.put("non_idempotent", ImmutableSet.copyOf(Lists.<StatusCode.Code>newArrayList()));
+      definitions.put("no_retry_codes", ImmutableSet.copyOf(Lists.<StatusCode.Code>newArrayList()));
```

Comment:
This PR is bad. Upon closer inspection of the grpc_service_config, there is a typo in the Service name, rendering the generator unable to assign the policies to the RPCs.

Reply:
This commit should fix it: https://github.com/googleapis/googleapis/commit/864dfeafdd5429e20612d378169858d9fe08c9e8